ACHDictionary.go

```diff
@@ -7,16 +7,28 @@ package fed
 import (
 	"bufio"
 	"github.com/moov-io/base"
+	"github.com/moov-io/fed/pkg/strcmp"
 	"io"
+	"sort"
 	"strings"
 	"unicode/utf8"
 )
 
+var (
+	// ACHJaroWinklerSimilarity is the search similarity percentage for strcmp.JaroWinkler for CustomerName
+	// (Financial Institution Name)
+	ACHJaroWinklerSimilarity = 0.85
+	// ACHLevenshteinSimilarity is the search similarity percentage for strcmp.Levenshtein for CustomerName
+	// (Financial Institution Name)
+	ACHLevenshteinSimilarity = 0.85
+	// ACHMaximumRecordsReturned is the maximum number of records to be returned - Currently this matches the FED Site
+	// maximum
+	ACHMaximumRecordsReturned = 499
```

Comment:
Were you thinking of supporting a `?limit=N` parameter on the HTTP api?

Reply:
So let me know.  If there is a use case to return more, lets go though it.